src/nvim/os/unix_defs.h

```diff
@@ -31,12 +31,6 @@
 #ifndef USR_EXRC_FILE
 # define USR_EXRC_FILE "~/.exrc"
 #endif
-#ifndef USR_VIMRC_FILE
-# define USR_VIMRC_FILE "~/.nvimrc"
```

Comment:
@justinmk Taking a look at the basedir spec again, most of the `XDG_` variables are defined relative to `$HOME`, which has its equivalent in `%UserProfile%` under Windows. I guess in that sense using that should be the correct behavior as per the basedir spec. However, `XDG_DATA_DIRS` and `XDG_CONFIG_DIRS` are undefined, since they default to paths Windows doesn't have.  

http://standards.freedesktop.org/basedir-spec/latest/ar01s03.html